lib/tmuxinator/config.rb

```diff
@@ -25,6 +25,10 @@ def home?
         File.directory?(home)
       end
 
+      def options_file
+        directory + ".yml"
```

Comment:
The "yaml" extension is also supported. I tried using a project config file with this extension and everything worked as expected. Does that imply that this method isn't being used or am I overlooking something?

Reply:
what do you suggest?

Reply:
Either this method needs to support both extensions or, if I'm correct in that it's not being used, it should be removed.